opensearch-operator/controllers/suite_test.go

```diff
@@ -109,7 +107,7 @@ var _ = BeforeSuite(func() {
 		Client: k8sManager.GetClient(),
 		Scheme: scheme.Scheme,
 		//	Instance: &OpensearchCluster,
-		Recorder: recorder,
+		Recorder: record.NewFakeRecorder(20),
```

Comment:
It creates a mock EventRecorder.  I have noticed in the tests that it flakes sometimes where it needs to record an event because the recorder wasn't instantiated so I added this to avoid that.